packages/cf-component-icon/src/Icon.js

```diff
@@ -1,33 +1,245 @@
+    case 'caret-down':
+    case 'chevron-down':
+      fontPoint = "'\\F001'";
+      break;
+    case 'caret-left':
+    case 'chevron-left':
+      fontPoint = "'\\F002'";
+      break;
+    case 'caret-right':
+    case 'chevron-right':
+      fontPoint = "'\\F003'";
+      break;
+    case 'ok-sign':
+      fontPoint = "'\\F004'";
+      break;
+    case 'exclamation-sign':
+      fontPoint = "'\\F005'";
+      break;
+    case 'info-sign':
+      fontPoint = "'\\F006'";
+      break;
+    case 'pause':
+      fontPoint = "'\\F007'";
+      break;
+    case 'time':
+      fontPoint = "'\\F008'";
+      break;
+    case 'ok':
+      fontPoint = "'\\F009'";
+      break;
+    case 'clipboard':
+      fontPoint = "'\\F00A'";
+      break;
+    case 'bolt':
+      fontPoint = "'\\F00B'";
+      break;
+    case 'chart':
+      fontPoint = "'\\F00C'";
+      break;
+    case 'drive':
+      fontPoint = "'\\F00D'";
+      break;
+    case 'filter':
+      fontPoint = "'\\F00E'";
+      break;
+    case 'flowchart':
+      fontPoint = "'\\F00F'";
+      break;
+    case 'hamburger':
+      fontPoint = "'\\F010'";
+      break;
+    case 'lock':
+      fontPoint = "'\\F011'";
+      break;
+    case 'list':
+      fontPoint = "'\\F012'";
+      break;
+    case 'file':
+      fontPoint = "'\\F013'";
+      break;
+    case 'shield':
+      fontPoint = "'\\F014'";
+      break;
+    case 'plus':
+      fontPoint = "'\\F015'";
+      break;
+    case 'happy':
+      fontPoint = "'\\F016'";
+      break;
+    case 'sad':
+      fontPoint = "'\\F017'";
+      break;
+    case 'search':
+      fontPoint = "'\\F018'";
+      break;
+    case 'wrench':
+      fontPoint = "'\\F019'";
+      break;
+    case 'remove-sign':
+    case 'remove':
+      fontPoint = "'\\F01A'";
+      break;
+    case 'speech':
+      fontPoint = "'\\F01B'";
+      break;
+    case 'gear':
+      fontPoint = "'\\F01C'";
+      break;
+    case 'help':
+      fontPoint = "'\\F01D'";
+      break;
+    case 'widen':
+      fontPoint = "'\\F022'";
+      break;
+    case 'calendar':
+      fontPoint = "'\\F01E'";
+      break;
+    case 'facebook':
+      fontPoint = "'\\F01F'";
+      break;
+    case 'twitter':
+      fontPoint = "'\\F020'";
+      break;
+    case 'googleplus':
+      fontPoint = "'\\F021'";
+      break;
+    case 'linkedin':
+      fontPoint = "'\\F022'";
+      break;
+    case 'resize-horizontal':
+      fontPoint = "'\\F024'";
+      break;
+    case 'reorder':
+    case 'resize-vertical':
+      fontPoint = "'\\F024'";
+      additionalRules.transform = 'rotate(90deg)';
+      break;
+    case 'upload':
+      fontPoint = "'\\F025'";
+      break;
+    case 'close':
+      fontPoint = "'\\D7'";
+      additionalRules.fontFamily = 'Open Sans, Helvetica, Arial, sans-serif';
+      break;
+  }
+
+  if (border) {
+    additionalRules.border = `1px solid ${color}`;
+    additionalRules.padding = '.2em .25em .15em';
+    additionalRules.borderRadius = '2px';
```

Comment:
Sounds good. What if this padding is specific to this component? Should we still abstract it out to a constant to be used (probably) in one place?